src/platforms/android/guides/okhttp/index.mdx

```diff
@@ -0,0 +1,37 @@
+<PlatformContent includePath="getting-started-primer" />
+
+On this page, we get you up and running with Sentry's OkHttp Integration, so that it will automatically add a breadcrumb and start a span out of the active span bound to the scope for each HTTP Request.
+
+<Note>
+
+If you don't already have an account and Sentry project established, head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+
+</Note>
+
+## Install
+
+Sentry captures data by adding an OkHttp `Interceptor`.
+
+<PlatformContent includePath="getting-started-install" />
+
+## Configure
+
+Configuration should happen once you create your `OkHttpClient` instance.
+
+<PlatformContent includePath="getting-started-config" />
+
+## Verify
+
+This snippet includes a HTTP Request and captures an intentional message, so you can test that everything is working as soon as you set it up:
+
+<PlatformContent includePath="getting-started-verify" />
+
+<Note>
+
+Learn more about manually capturing an error or message, in our <PlatformLink to="/usage/">Usage documentation</PlatformLink>.
+
+</Note>
+
+To view and resolve the recorded error, log into [sentry.io](https://sentry.io) and open your project. Clicking on the error's title will open a page where you can see detailed information and mark it as resolved.
```

Comment:
`l`: Maybe we could be a bit more specific here. It is not a recorded error but a message and it would be nice to explain where in the issue I have to go to see the breadcrumbs and the spans.

Reply:
changed `error` to `message`

> Clicking on the error's title will open a page where you can see detailed information and mark it as resolved.

breadcrumbs and traces associated with the event can be seen from that page